week_1/Project/youtube-fe/youtube.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube</title>
+    <link rel="stylesheet" href="styles/general.css">
+    <link rel="stylesheet" href="styles/header.css">
+    <link rel="stylesheet" href="styles/sidebar.css">
+    <link rel="stylesheet" href="styles/video.css">
+</head>
+
+<body>
+    <div class="header">
+        <div class="left-section">
+            <img class="hamburger-menu" src="icons/hamburger-menu.svg" alt="hamburger">
+            <img class="youtube-logo" src="icons/youtube-logo.svg" alt="youtube">
+        </div>
+        <div class="middle-section">
+            <input class="search-bar" type="text" placeholder="Search">
```

Comment:
@codal-kcolaco Please check all input's value and how they all are used and why?

Reply:
@abhi-codal  Yes will do.